lineage/figures/figureCommon.py

```diff
@@ -391,12 +391,12 @@ def plotting(ax, lpt_avg, bern_lpt, cons, concsValues, num_states):
     # ylim and ylabel
     for i in range(10, 12):
         ax[i].set_ylabel("log10 mean time [hr]")
-        ax[i].set_ylim([0, 8.0])
+        ax[i].set_ylim([0, 3.5])
 
     # ylim and ylabel
     for i in range(12, 14):
         ax[i].set_ylabel("division probability")
-        ax[i].set_ylim([0, 1.05])
+        ax[i].set_ylim([0.25, 1.05])
```

Comment:
Please do not change h/i. It's useful to have probability on a scale of 0–1, and these can be read just fine.

Reply:
done,
![image](https://user-images.githubusercontent.com/34812750/126369130-1ac51702-d654-4d56-8ede-71cae4f02b18.png)

![image](https://user-images.githubusercontent.com/34812750/126369157-b01f82c0-11f1-4641-a53a-d3b58d6b503e.png)